render.c

```diff
@@ -168,16 +168,7 @@ void render_frame(struct swaylock_surface *surface) {
 			xkb_layout_index_t num_layout = xkb_keymap_num_layouts(state->xkb.keymap);
 			if (!state->args.hide_keyboard_layout &&
 					(state->args.show_keyboard_layout || num_layout > 1)) {
```

Comment:
Also, maybe I should add `&& state->xkb.layout_set` here?

Is it possible that this code will be executed before current keyboard layout is set by keyboard_modifiers()?